assets/src/dashboard/app/views/storyAnimTool/components.js

```diff
@@ -76,10 +76,8 @@ export const ElementInfo = styled.button(
     width: 100%;
     text-align: left;
     border: none;
-    border-bottom: 1px solid ${theme.DEPRECATED_THEME.colors.gray500};
-    background-color: ${
-      isActive ? theme.DEPRECATED_THEME.colors.gray75 : 'transparent'
-    };
+    border-bottom: 1px solid ${theme.colors.border.defaultNormal};
+    background-color: ${isActive ? theme.colors.bg.secondary : 'transparent'};
```

Comment:
this is fine, but just so you know there's a `colors.opacity.footprint` which is transparent.